reductstore/src/api/bucket/get.rs

```diff
@@ -19,9 +19,9 @@ pub(crate) async fn get_bucket(
     check_permissions(&components, headers, AuthenticatedPolicy {}).await?;
     let bucket_info = components
         .storage
-        .read()
+        .write()
```

Comment:
looks like a big change from read -> to write. I'm sure you have a reason, but just to double check

Reply:
Yes, this is because of block cache.. Even for read operation we change a block in cache, that it is why we need to write here. I have an idea how to do it better but I'd like to do it in the next release. To much refactoring already.